pilot/pkg/security/trustdomain/bundle.go

```diff
@@ -26,21 +26,22 @@ var (
 )
 
 type Bundle struct {
+	// Contain the local trust domain and its aliases.
 	// The trust domain corresponds to the trust root of a system.
 	// Refer to [SPIFFE-ID](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain)
-	TrustDomain string
 	// The trust domain aliases represent the aliases of `trust_domain`.
 	// For example, if we have
 	// trustDomain: td1, trustDomainAliases: ["td2", "td3"]
 	// Any service with the identity `td1/ns/foo/sa/a-service-account`, `td2/ns/foo/sa/a-service-account`,
 	// or `td3/ns/foo/sa/a-service-account` will be treated the same in the Istio mesh.
-	TrustDomainAliases []string
+	TrustDomains []string
 }
 
 func NewTrustDomainBundle(trustDomain string, trustDomainAliases []string) Bundle {
+
 	return Bundle{
-		TrustDomain:        trustDomain,
-		TrustDomainAliases: trustDomainAliases,
+		// Put the new trust domain to the beginning of the list to avoid changing existing tests.
```

Comment:
Is this only for test or is it a required property for this data structure?

Reply:
Mostly just for tests. Most tests are written with the new trust domain to appear first.